samples/ps2dev_iop.cmake

```diff
@@ -0,0 +1,118 @@
+#
+# CMake platform file for PS2 IOP processor
+#
+# Copyright (C) 2009-2010 Mathias Lafeldt <misfire@debugon.org>
+# Copyright (C) 2023 Francisco Javier Trujillo Mata <fjtrujy@gmail.com>
+# Copyright (C) 2024 André Guilherme <andregui17@outlook.com>
+# Copyright (C) 2024-Present PS2DEV Team
+#
+
+INCLUDE(CMakeForceCompiler)
+if(DEFINED ENV{PS2SDK})
+    SET(PS2SDK $ENV{PS2SDK})
+else()
+    message(FATAL_ERROR "The environment variable PS2SDK needs to be defined.")
+endif()
+
+if(DEFINED ENV{PS2DEV})
+    SET(PS2DEV $ENV{PS2DEV})
+else()
+    message(FATAL_ERROR "The environment variable PS2DEV needs to be defined.")
+endif()
+
+SET(CMAKE_SYSTEM_NAME Generic)
+SET(CMAKE_SYSTEM_VERSION 1)
+SET(CMAKE_SYSTEM_PROCESSOR mips)
+SET(CMAKE_C_COMPILER mipsel-ps2-irx-gcc)
+SET(CMAKE_CXX_COMPILER mipsel-ps2-irx-g++)
+SET(CMAKE_C_COMPILER_WORKS 1) #Hack by f0bes
+SET(CMAKE_CXX_COMPILER_WORKS 1) #Hack by f0bes
+
+#
+# set arch flags depending on gcc version
+#
+EXECUTE_PROCESS(COMMAND ${CMAKE_C_COMPILER} -dumpversion
+    OUTPUT_VARIABLE IOP_CC_VERSION)
+
+SET(IOP_ASFLAGS_TARGET "-mcpu=r3000")
+
+IF (${IOP_CC_VERSION} VERSION_EQUAL 3.2.2)
```

Comment:
we are not using anymore legacy toolchain, so remove this piece of code

Reply:
Then i should soon create a pull request that removes old acient gcc code on makefile